server/src/modules/users/domain/value-objects/auth-code-string.ts

```diff
@@ -1,25 +1,25 @@
 import crypto from 'crypto'
 
 export class AuthCodeString {
-  private value: string;
-  
+  private value: string
+
   public constructor(hashedValue?: string) {
-    if(hashedValue){
+    if (hashedValue) {
```

Comment:
you might run into truthiness problems with empty strings

```suggestion
    if (hashedValue !== undefined) {
```